pkg/services/store/entity/server/service.go

```diff
@@ -64,6 +64,7 @@ func ProvideService(
 	if err != nil {
 		return nil, err
 	}
+	tracingCfg.ServiceName = "unified-storage"
```

Comment:
when were running this as it's own service, we overwrite the default "grafana" trace service name